CONTRIBUTING.md

```diff
@@ -326,6 +326,31 @@ Also, make sure your image is compressed as much as possible
 This can be done with [pingo][]'s lossless PNG compression:
 `pingo -sa file.png`.
 
+<!-- TODO: put this in a infobox? -->
+When extracting frames directly from a VapourSynth pipline
+where the format might be `vs.YUV420P16` (YUV 4:2:0, 16-bit),
+convert your image to `vs.RGB24` (RGB 8-bit) before saving as a PNG.[^3]
+This is because many, if not all, browsers don't support
+images with bit-depths higher than 8 bpp,
+and the dithering behavior of some browsers may be different from others
+or poorly executed.
+If you are trying to replicate media player chroma behavior
+in your screenshot or comparison,
+upscale the chroma planes using:
+
+```py
+clip = core.resize.Bicubic(clip, clip.width, clip.height, filter_param_a_uv=0.33, filter_param_b_uv=0.33, format=clip.format.replace(subsampling_w=0, subsampling_h=0))
+```
+
+You can change the format and bit-depth
+while saving to a PNG file with the following lines:
+
+```py
+# replace `{frame}` with the frame number of the clip you are extracting
+out = core.imwri.Write(clip[{frame}].resize.Point(format=vs.RGB24, matrix_in_s='709', range=0, range_in=0, dither_type='error_diffusion'), 'PNG', '%06d.png', firstnum={frame})
```

Comment:
> range_in=0

YCbCr content is usually limited range. The range parameters can be omitted in the most likely case of limited input. Also, the chroma upscaling and RGB conversion can be condensed into just one call:
```py
clip.resize.Bicubic(format=vs.RGB24, matrix_in_s='709', dither_type='error_diffusion', filter_param_a_uv=0.33, filter_param_b_uv=0.33)
```